server.js

```diff
@@ -41,6 +42,7 @@ app.engine('html', hbs.express4({
 // Middleware
 app.set('view engine', 'html')
 app.set('views', path.join(__dirname, '/views'))
+app.use(nakedRedirect(true, 'www', 301))
```

Comment:
Can I request we make this `302` instead of `301` for a few days at least to ensure we aren't redirecting badly.  I'm always scared of `301` because there is no way to undo any damage done by it 😆 **Permanent**